configure

```diff
@@ -69,6 +70,7 @@ Options:
   -no-runtime-lib      Do not compile nor install the runtime support library
   -no-standard-headers Do not install nor use the standard .h headers
   -clightgen           Also compile the clightgen tool
+  -ignore-coq-version  Do not check the version of Coq
```

Comment:
Waiting on feedback from Comcert folks. I suggest you add the 8.7 to Coq's 8.6 travis.